src/content/docs/apis/nerdgraph/examples/nerdgraph-issues-api-via-github.mdx

```diff
@@ -0,0 +1,2103 @@
+---
+title: Issues API using Nerdgraph
+metaDescription: "Set capabilities and entitlements in Detection and AI"
+redirects: 
+---
+
+This doc is a place for you and your team to learn how to query your data for issues and incidents and then configure features using our NerdGraph API.
+
+Our two main data types are `Issue Type` and `Incident Type`. 
+
+## Issue Type [#issue-type]
```

Comment:
I"m not sure if this list of metadata maps over to docs we already have in alerts and AI space. Do we already list these somewhere in alerts and AI docs? Can we tie those two together in some way? Is it possible this information is redundant with what we have there in some way?